releases/build.sh

```diff
@@ -35,7 +35,10 @@ fi
 
 
 # Build the Docker image (this can take a while):
-docker build --pull --force-rm --no-cache -t bitbox02-firmware .
+#  The firmware is only reproducible with the same host compiler. For now we force linux/amd64.
+#  The file .containerversion does not exist in older versions of the project.
+containerversion=$(2>/dev/null cat .containerversion || echo "latest")
+docker build --pull --platform linux/amd64 --force-rm --no-cache -t bitbox02-firmware:${containerversion} .
```

Comment:
you don't need to tag anything here, it serves no purpose. the image is built here from scratch and used in the next line, not pulled, so the local image name/tag is irrelevant, and using `.containerversion` here is distracting/confusing, as it is unrelated to our shiftcrypto/firmware_v2 tagging.

Reply:
no problem, fixed!